pkg/services/contexthandler/contexthandler.go

```diff
@@ -105,6 +106,19 @@ func (h *ContextHandler) Middleware(mContext *web.Context) {
 		}
 	}
 
+	queryParameters, err := url.ParseQuery(reqContext.Req.URL.RawQuery)
+	if err == nil {
+		reqContext.Logger.Error("Failed to parse query parameters", "error", err)
+	}
+	if queryParameters.Has("targetOrgId") {
+		targetOrg, err := strconv.ParseInt(queryParameters.Get("targetOrgId"), 10, 64)
```

Comment:
I was going to suggest `Atoi` to simplify things, but that'd require an extra conversion from int to int64, so I'm not going to suggest that. Seems like "flip a coin" would be a tolerable way to choose which of those two you were going with here.

Reply:
I did actually initially use `Atoi`, but then saw that we do it using `ParseInt` higher up (if `X-Grafana-Org-ID` header is set), so decided to keep it consistent.